### ISSUE TITLE
Patchouli (essentials/crossroads) book text getting replaced by emoji

### ISSUE THREAD

**Issue by CvoidWyvern** [InnovativeOnlineIndustries/Emojiful#5]
https://cdn.discordapp.com/attachments/350295690246881288/759861106625216533/2020-09-27_14.31.56.png

This specific instance seems to also involve emojiful using :1: as 👍  instead of :+1:

On 1.16.1-2.0.1, forge 32.0.108

**Comment by ChipWolf** [comment]
I'm guessing we'll have to store emojis in the book rather than replacing the markup on render to fix this.

**Comment by Buuz135** [comment]
Does still happen?

**Comment by CvoidWyvern** [comment]
Yes, still occurs with 2.1.2, crossroads 2.6.0, essentials 2.10.0, patchouli 48, forge 35.1.29
